rust/osm2lanes/src/tag/mod.rs

```diff
@@ -90,6 +92,27 @@ impl Tags {
         self.map.get(k.as_ref()).map(String::as_str)
     }
 
+    /// Get the value for the given key and parse it into T.
+    /// Add a `RoadMsg::Unsupported` if parsing fails.
+    // TODO: check if interface should be moved to transform
```

Comment:
Yes, `get_parsed` exists for the convenience of things like `let tagged_lanes: Option<usize> = tags.get_parsed(&LANES, warnings)`, and should live in the `transform` module somewhere.